src/protocolsupport/protocol/packet/middleimpl/clientbound/play/v_pe/StartGame.java

```diff
@@ -110,9 +110,10 @@ public StartGame(ConnectionImpl connection) {
 		VarNumberSerializer.writeSVarInt(chunkradius, (int) Math.ceil((Bukkit.getViewDistance() + 1) * Math.sqrt(2)));
 		packets.add(chunkradius);
 		//Send crafting recipes
-//		ClientBoundPacketData craftPacket = ClientBoundPacketData.create(PEPacketIDs.CRAFTING_DATA); 
-//		craftPacket.writeBytes(PECraftingManager.getInstance().getAllRecipes());
-//		packets.add(craftPacket);
+		//TODO: bungeecord should also request this from servers again
```

Comment:
Maybe not. 1.13 sends crafting recipes, making the whole cached thing outdated since plugins etc will be able to register recipes on the fly and multiple servers can have different crafting recipes. That's why the system was commented out.